GitUpKit/Core/GCFoundation.m

```diff
@@ -19,10 +19,19 @@
 
 #import "GCPrivate.h"
 
-#if !TARGET_OS_IPHONE
-
 @implementation NSFileManager (GCFoundation)
 
+- (BOOL)fileExistsAtPath:(NSString*)path followSymlink:(BOOL)followSymlink
+{
+  if (followSymlink) {
+    return [self fileExistsAtPath:path];
+  }
+
+  return [self attributesOfItemAtPath:path error:NULL] != nil;
```

Comment:
I'd rather use a single API rather than 2. Looking at the docs, something like that would work:
```
if (followSymlink) {
  path = [path stringByResolvingSymlinksInPath];
}
return [self attributesOfItemAtPath:path error:NULL] != nil;
```